mibidata/tiff.py

```diff
@@ -14,7 +14,7 @@
 from mibidata import mibi_image as mi, util
 
 # Increment this when making functional changes.
-SOFTWARE_VERSION = 'IonpathMIBIv0.2'
+SOFTWARE_VERSION = 'IonpathMIBIv1.0'
```

Comment:
Should we get this from the MibiImage rather than hard-coding it in two places?

Reply:
@rfinck I found that in the TIFF creator file, and wasn't sure about the purpose of it. I would also think that if we make a MibiImage version, the version of the TIFF should match it.  So, yes, I will definitely make a check on the MibiImage version and build the the software version accordingly. Or should it be exactly the same?
Alternatively, we can create a dedicated python file called mibitiff_version.py with the version number and methods to extract and compare version numbers and make all pieces of the software that need the version read it from there.

Reply:
The TIFFs we make need to be compatible with various third-party image analysis software software, and some of them use this tag to determine which reader to use. This includes but is not limited to BioFormats (general info [here](https://docs.openmicroscopy.org/bio-formats/5.8.2/formats/ionpath-mibi.html), code that uses version string [here](https://github.com/ome/bioformats/blob/develop/components/formats-gpl/src/loci/formats/in/IonpathMIBITiffReader.java#L54). So the MIBItiff version is a specific thing that has use and meaning outside of the library version. For now, let's not expand scope of this ticket and get the 1.0 version number in here in the simplest way that doesn't require it to be maintained separately from the number you've written into MibiImage (e.g. constructing this string by reading the number from there) and we can do other version management updates in a follow-up PR.

Reply:
Ok, I made the software version dependent on the MIBItiff version.